src/pages/List.js

```diff
@@ -1,3 +1,28 @@
+import { db } from '../lib/firebase';
+import { useCollection } from 'react-firebase-hooks/firestore';
+
 export default function List(props) {
-  return <h1>THIS IS THE LIST</h1>;
+  const [value, loading, error] = useCollection(
```

Comment:
Consider using a more meaningful name for shopping list items other than `value' as you are building this out.

Reply:
Thanks Yenly! This is a great suggestion, because value doesnt describe exactly what it is, so I am going to change `value` to `listItem`